libs/domains/service-logs/feature/project.json

```diff
@@ -4,6 +4,7 @@
   "sourceRoot": "libs/domains/service-logs/feature/src",
   "projectType": "library",
   "tags": ["scope:domain", "type:feature", "slice:service-logs"],
+  "implicitDependencies": ["!shared-console-shared"],
```

Comment:
Unnecessary for this library

Reply:
https://github.com/Qovery/console/pull/1683/commits/c719204017237ce9e808e3185d670db37d2d463b